src/Forms/MultiLineTextInput.tsx

```diff
@@ -0,0 +1,110 @@
+import React, {
+  ReactElement,
+  FunctionComponent,
+  ChangeEventHandler,
+  useContext,
+  Ref,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { fromTheme } from '../Theme';
+import InputLabel, { InputLabelPosition, POSITION } from './InputLabel';
+import ValidationErrorMessage from './ValidationErrorMessage';
+
+export interface MultiLineTextInputProps {
+  /** The id of the label tied to this textarea */
+  id: string;
+  /** The body of the error message, if applicable */
+  errorMessage?: string;
+  /** The value of the multi-line text area */
+  value?: string;
+  /** Function to call on change event */
+  onChange: ChangeEventHandler;
+  /** Used to control the suppression of the error message value */
+  hideError?: boolean;
+  /** Marks the field as required */
+  isRequired?: boolean;
+  /** The label to display beside this multi-line input field(if any) */
+  label?: string;
```

Comment:
I believe the `label` should be a required prop for accessibility.

Reply:
Additionally I think that we need the `name` property since this will be part of a form, and in the case that there is no label, I think it could be useful to also have the `placeholder` property exposed:
![image](https://user-images.githubusercontent.com/29589689/152045074-85be7eb0-4a67-4636-9180-8c228e1abe4b.png)

Reply:
Yes, it should. Fixed in 0ff2505e7ed4b8513fcfadb85131f5c4d3a1afb6

Reply:
> Additionally I think that we need the `name` property since this will be part of a form, and in the case that there is no label, I think it could be useful to also have the `placeholder` property exposed: ![image](https://user-images.githubusercontent.com/29589689/152045074-85be7eb0-4a67-4636-9180-8c228e1abe4b.png)

D'oh. I knew I'd forgotten something. Yeah, we need the `name` prop. Added that in 9b57d72


As for the `placeholder` prop - I also agree. Fixed that in be3eed4b55c92ff4810b51159d7e2c52509e5fa8